iptables/iptables.go

```diff
@@ -256,7 +242,11 @@ func executeCommand(firewallConfiguration FirewallConfiguration, cmd *exec.Cmd,
 	out, err := cmd.CombinedOutput()
 
 	if len(out) > 0 {
-		log.Printf("%s\n", out)
```

Comment:
Similarly, I think we should log the full output here at INFO level, regardless of whether there is an err.